.rubocop-https---relaxed-ruby-style-rubocop-yml

```diff
@@ -0,0 +1,166 @@
+# Relaxed.Ruby.Style
```

Comment:
I think this file should live under 

`lib/solidus_cmd/templates/extension`

no?